lavalink/client.py

```diff
@@ -47,35 +48,14 @@ def __init__(self, user_id: int, shard_count: int = 1,
         self._shard_count = str(shard_count)
         self._loop = loop or asyncio.get_event_loop()
         self.node_manager = NodeManager(self, regions)
-        self.players = PlayerManager(self, player)
+        self.player_manager = PlayerManager(self, player)
```

Comment:
why

Reply:
Do you really want `Client.players.players`

Reply:
Well it's a breaking change then

Reply:
I need to change the example, I forgot to do that.

Reply:
I'm okay with breaking changes as the hook thing is a breaking change. I'll just bump the minor version. (I should follow semver (i.e. bump major version) but I really do not feel like this warrants `Lavalink.py v4`)

Reply:
What about version `3.1.0`

Reply:
That's what I meant by minor version lol